src/OidcClient.php

```diff
@@ -99,11 +103,29 @@ public function refreshTokens(string $refreshToken): OidcTokens
 
     // Request and verify the tokens
     return $this->verifyTokens(
```

Comment:
And we make the `verifyTokens` class return the `OidcTokens` class, created from the `UnvalidatedOidcTokens`

Reply:
:heavy_check_mark: